bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -202,6 +218,18 @@
 		<description>DC voltage</description>
 		<state pattern="%.1f V" readOnly="true"></state>
 	</channel-type>
+	<channel-type id="udc2">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC Voltage 2</label>
+		<description>DC voltage</description>
+		<state pattern="%.1f V" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="udc3">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC Voltage</label>
```

Comment:
As already noticed by @curlyel:
```suggestion
		<label>DC Voltage 3</label>
```

Reply:
Thanks for spotting this. I have updated the label.